src/com/goide/GoFileType.java

```diff
@@ -62,8 +62,4 @@ public String getCharset(@NotNull VirtualFile file, @NotNull byte[] content) {
     return CharsetToolkit.UTF8;
   }
 
-  @Override
-  public Charset extractCharsetFromFileContent(@Nullable Project project, @Nullable VirtualFile file, @NotNull CharSequence content) {
```

Comment:
Why remove this from the Go files? They should always be UTF-8 per language specs (or do I miss something?)